acceptance-tests/tests/build.gradle

```diff
@@ -24,6 +24,7 @@ sourceSets.main.solidity.srcDirs = ["$projectDir/contracts"]
 
 solidity {
   resolvePackages = false
+  executable = "/usr/bin/solc"
```

Comment:
I think we should do something like:

`executable = System.getenv('SOLC_BINARY_PATH') ?: null`

then if you don't have solc installed sokt will attempt to fetch it for you (if services are running)

Reply:
for running it locally?

Reply:
In an ideal world, I'd be able to clone besu and run `./gradlew check` and not have to have anything installed. 

By adding the path explicitly it means that everyone needs to have solc installed in the same place to be able to build. It also means that you can only build on *nix! 

Another option would be to put some code into the gradle scripts to download solc if it's needed (basically doing what sokt does at the moment)

Reply:
the build already requires some packages to be installed, libsodium and nssdb, so adding another is not changing to much, and running tests on Windows does not work yet, but I agree that these are shortcomings that could be addressed.